Assignment4/josellorentev-190021/task06.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/105RNbZdDPid-La5HphhUvLXM3l9nYj8i
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2021-2022/master/Assignment4/course_materials"
+
+"""Leemos el fichero RDF de la forma que lo hemos venido haciendo"""
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example5.rdf", format="xml")
+
+"""Create a new class named Researcher"""
+
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.1: Create a new class named "University"**
+
+"""
+
+# TO DO
+ns = Namespace("http://somewhere#")
+g.add((ns.University, RDF.type, RDFS.Class))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+
+# TO DO
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+
+# TO DO
+g.add((ns.JaneSmith, RDF.type, ns.Researcher))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**"""
+
+# TO DO
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.JaneSmith, vcard.FN, Literal("Jane Smith")))
+g.add((ns.JaneSmith, vcard.Given, Literal("Jane")))
+g.add((ns.JaneSmith, vcard.Family, Literal("Smith")))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+ns = Namespace("http://somewhere#")
+g.add((ns.Ump, RDF.type, ns.University))
+g.add((ns.JohnSmith, vcard.work, ns.UPM))
```

Comment:
work is not defined in vcard. Define a new property on your namespace

Reply:
i have done the changes already and i have done the first assigment of the proyect, i think that both merge in one pull request. Do i need close one and do two separately request or with this one it is correct?

Reply:
should be ok.

Reply:
This is still not done. You are still defining the work property as a property in vcard